lib/login.c

```diff
@@ -1220,7 +1220,8 @@ iscsi_process_login_reply(struct iscsi_context *iscsi, struct iscsi_pdu *pdu,
 				 * the target said. ImmediateData is NO.
 				 */
 				iscsi->use_immediate_data = ISCSI_IMMEDIATE_DATA_NO;
-
+			} else if (iscsi->use_immediate_data == ISCSI_IMMEDIATE_DATA_NO) {
```

Comment:
Hmm ... do I understand correctly that if use_immediate_data == ISCSI_IMMEDIATE_DATA_NO it remains ISCSI_IMMEDIATE_DATA_NO and if use_immediate_data == ISCSI_IMMEDIATE_DATA_YES it remains ISCSI_IMMEDIATE_DATA_YES? Does this mean that these four lines can be deleted?

Please also make the following changes:
* In the commit title, change "Fix to lose" into "Retain".
* Fix the commit author with git commit --amend --reset-author. The current author information (`root <root@dev-ssamoylenko.spb.yadro.com>`) is not valid since it does not include the name of a person.

Reply:
You are right. This lines can be deleted and it will be the same.